packages/components/src/Link/Link.stories.tsx

```diff
@@ -0,0 +1,184 @@
+import { Story } from "@storybook/react/types-6-0";
+import React from "react";
+import Clickable from "../Clickable";
+import Heading from "../Heading";
+import CheckCircleRoundedIcon from "../Icons/CheckCircleRounded";
+import Text from "../Text";
+import Link, { LinkProps } from "./Link";
+import styles from "./Link.stories.module.css";
+
+const sizes = ["small", "medium", "large"] as const;
+const allColors = ["alert", "brand", "neutral", "success", "warning"] as const;
+const variants = ["flat", "outline", "link"] as const;
+const states = ["inactive", "hover", "focus", "active", "disabled"] as const;
+
+// For now, the UI kit only includes alert & brand "flat" buttons.
+const flatColors = ["alert", "brand"] as const;
+
+export default {
+  title: "Link",
+  component: Link,
+  argTypes: {
+    children: {
+      control: {
+        type: "text",
+      },
+    },
+    color: {
+      control: {
+        type: "radio",
+        options: allColors,
+      },
+    },
+  },
+};
+
+type Args = React.ComponentProps<typeof Link>;
+
+const Template: Story<Args> = (args) => <Link {...args} />;
+
+// TODO: Consolidate with button stories.
+
+const defaultArgs = {
+  children: "Link",
+  variant: "link" as const,
+  color: "brand" as const,
+  href: "",
+};
+
+export const StandardLink = Template.bind(null);
+StandardLink.args = {
+  ...defaultArgs,
+};
+
+export const linkInBody: Story<Args> = (args) => (
+  <Text size="body">
+    This text surrounds the <Link {...args} /> and shows that the link should
+    adhere to its appearance
+  </Text>
+);
+linkInBody.args = {
+  ...defaultArgs,
+};
+
+export const linkInHeading: Story<Args> = (args) => (
+  <Text size="h1">
+    This text surrounds the <Link {...args} /> and shows that the link should
+    adhere to its appearance
+  </Text>
+);
+linkInHeading.args = {
+  ...defaultArgs,
+};
+
+// Show grids with all variants
+
+const gridParameters = {
+  axe: {
+    skip: true,
+  },
+  snapshot: {
+    skip: true,
+  },
+};
+
+const renderSize = (
+  size: LinkProps["size"],
+  textColor: "white" | "neutral",
+  children: React.ReactNode,
+) =>
+  variants.map((variant) => {
+    const colors = variant === "flat" ? flatColors : allColors;
+
+    return (
+      <React.Fragment key={variant}>
+        <Heading size="h2" color={textColor}>
+          {variant} - {size}
+        </Heading>
+        <table className={styles.variant}>
+          <tbody>
+            {states.map((state) => (
+              <tr key={state}>
+                <th scope="row">
+                  <Text size="body" color={textColor}>
+                    {state}
+                  </Text>
+                </th>
+                {colors.map((color) => (
+                  <td key={color} className={styles.color}>
```

Comment:
Would be nice to directly use Tailwind classes, instead of having to create a CSS file.

Could be interesting at some point to experiment with integrating Tailwind (ideally in [JIT mode](https://tailwindcss.com/docs/just-in-time-mode)) into our Storybook.